### ISSUE TITLE
Implement bodhi overrides --mine

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1382]
Zbigniew Jędrzejewski-Szmek [filed an issue in Red Hat's Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1218042) requesting the bodhi client to support a ```--mine``` flag as a shortcut to express ```--user <username>```.

I.e., I could search for my overrides like this:

```
$ bodhi overrides query --mine
```

instead of:

```
$ bodhi overrides query --user bowlofeggs
```

**Comment by nbianca** [comment]
Hello @bowlofeggs,

If option `--mine` is used what should be the default username? `$USER`? I have looked through the source code and saw that you can use env var `USERNAME` to avoid typing `--user` every time.

**Comment by bowlofeggs** [comment]
On Thu, 2017-03-23 at 15:33 -0700, Bianca Nenciu wrote:
> If option --mine is used what should be the default username? $USER?
> I have looked through the source code and saw that you can use env
> var USERNAME to avoid typing --user every time.

Hi Bianca!

Yes, using the USERNAME environment variable is probably a reasonable
way to do this. When this is done, it should be documented in the man
page and it would be good to mention that the environment variable is
required for it to operate correctly.

I noticed that the USERNAME env var wasn't documented in the man page
so I added that here:

https://github.com/fedora-infra/bodhi/pull/1384
